source/slang/slang-ir-lower-buffer-element-type.cpp

```diff
@@ -319,6 +381,15 @@ struct LoweredElementTypeContext
         }
     }
 
+    IRIntegerValue get16ByteAlignedVectorSize(IRType* elementType, IRIntegerValue minCount)
```

Comment:
Does this function actually return the number of elements after alignment instead of the size?

Reply:
yeah, by `VectorSize`, I really just mean the number elements. Will rename it to VectorElementCount.